README.md

```diff
@@ -43,7 +43,7 @@ To run this sample, you'll need:
 > - [Python 2.7+](https://www.python.org/downloads/release/python-2713/) or [Python 3+](https://www.python.org/downloads/release/python-364/)
 > - MSAL Python
 >   ```Python
->   pip msal
+>   pip install -r requirements.txt
 >   ```

```

Comment:
Good catch on that typo.

On a closer look, the root folder of this repo contains no `requirements.txt` at all. There are two in the two sub-folders, and their corresponding README files are correct. (This kind of chapterized sample is not as straightforward as two standalone samples in this regard.)

Shall we simply remove this entire "MSAL Python" bullet point from this root README? If you are OK with that, I can proceed to make the change for us. Let me know.